packages/core/src/types/provider.ts

```diff
@@ -405,3 +407,18 @@ export type PlaybackInfo = {
     }[];
   };
 };
+
+export type GetMetricsArgs = {
+  assetId: string;
+
+  // by: "day" |
+};
+
+export type Metric = {
```

Comment:
I think this should be either `AsetMetric` or `ViewershipMetric` (depending on what the `getMetrics` function becomes)

Reply:
Wouldn't this be used for streams as well?

Reply:
Yeah! But we might have other metric types as well, not related to viewership directly. For streams we have the stream health stuff for example.

So I think `ViewsMetric` or `ViewershipMetric` would be more accurate if we want a more future-proof name.

Reply:
As for specifying `Asset*` or not, not strong opinion. While we will have stream metrics in the future, we don't really have those now and the shape of the metrics might look very different anyway. So naming `Asset*Metric` now with the risk of it becoming just `*Metric` (if we realize all metrics are the same) seems relatively similar to naming it just `*Metric` now with the risk of having to make it `Asset*Metric` in the future (if we realize the metrics are different).